Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+
+gem 'rubocop', require: false
```

Comment:
Shall we go for double quotes here? 🙈

Reply:
Yes. Also, we use Rubocop for linting checks and using single quotes, in that case, is the recommended way.

Reply:
Oh, I was referring to `gem "fastlane"` actually, to have it like this instead: `gem 'fastlane'`

But no biggie, can leave it like it is

Reply:
Ah I see. I will update it. It doesn't look right at all.